session/session.go

```diff
@@ -147,7 +135,7 @@ func (s *ZKSession) manage() {
 			case zookeeper.STATE_EXPIRED_SESSION:
 				s.log.Printf("gozk-recipes/session: got STATE_EXPIRED_SESSION for conn %+v", s.conn)
 				expired = true
-				conn, events, err := zookeeper.Redial(s.servers, s.recvTimeout, s.clientID)
+				conn, events, err := zookeeper.Redial(strings.Join(s.opts.servers, ","), s.opts.recvTimeout, s.opts.clientID)
```

Comment:
Why the switch to the `strings.Join(...)` usage?

Reply:
zookeeper.Redial takes comma separates lists, so I have to join the array.